src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/WebLoginTests.kt

```diff
@@ -0,0 +1,31 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
+
+import org.assertj.core.api.Assertions
+import org.junit.Before
+import org.junit.Test
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.vaccineimpact.orderlyweb.db.AppConfig
+
+class WebLoginTests: SeleniumTest()
```

Comment:
I'm not really sure what the point of this file is, or why it's separate from GithubWebTests. `can login with github` is [already tested](https://github.com/vimc/orderly-web/blob/9a1c4efdfb56e7dc95df14bfd3a9fa34d260bc62/src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt#L24) in that file, all this test is doing additionally is adding a check for the link text?

To test the change that you've made, what it really should be doing is checking the link url, and making sure it includes the whole appUrl (not just relative link) - can check the link text too I guess since that does seem to be a missing test. But I'm really not sure why you think that test should go in a separate file, since GithubWebTest is already testing this login page and functionality.

Reply:
I think GithubWebTests should be renamed to WebLoginTests thou but I have moved the test for now.

Reply:
That would make sense to match the template, though I'm not sure the template is all that well named tbh! Effectively we only (currently) ever use it for Github login. GithubLoginTests or OAuthLoginTests maybe..